src/codegen/tests/expression_tests.rs

```diff
@@ -518,48 +518,6 @@ fn compare_date_time_literals() {
     insta::assert_snapshot!(result);
 }
 
-#[test]
-fn hardware_access_codegen() {
-    let result = codegen(
-        "
-        PROGRAM prg
-        VAR
-          x,y,z : BYTE;
-        END_VAR
-          x := %IB1.2;
-          y := %MB1.2;
-          z := %GB1.2;
-          x := %IX1.2;
-          y := %MD1.2;
-          z := %GW1.2;
-        END_PROGRAM
-        ",
-    );
-
-    insta::assert_snapshot!(result);
-}
-
-#[test]
-fn hardware_access_assign_codegen() {
-    let result = codegen(
-        "
-        PROGRAM prg
-        VAR
-          x,y,z : BYTE;
-        END_VAR
-          %IB1.2 := 1;
-          %MB1.2 := 1;
-          %GB1.2 := 1;
-          %IX1.2 := 1;
-          %MD1.2 := 1;
-          %GW1.2 := 1;
-        END_PROGRAM
-        ",
-    );
-
-    insta::assert_snapshot!(result);
-}
-
```

Comment:
You've deleted tests here, but not the matching snapshots

Reply:
i'll delete all snaps and regenerate makes it easier